### ISSUE TITLE
log messages should not be formatted

### ISSUE THREAD

**Issue by unicornware** [rollup/rollup#5699]
### Expected Behavior / Situation

I am building a tool on top of rollup. I'm using `onLog` to intercept logs because I want to format them differently. I expected log messages to **not** already be formatted.

### Actual Behavior / Situation

Rollup [prepends additional info to `log.message`][augment-log-message]. This makes for extra work if one **only** wants the original message text.

[augment-log-message]: https://github.com/rollup/rollup/blob/v4.24.0/src/utils/logs.ts#L81-L100

### Modification Proposal

Any rollup-specific formatting should be applied once the default log handler is called. If this isn't possible for some reason, a `text` property containing the original message should be added to `RollupLog`.

**Comment by lukastaegert** [comment]
Yes, I totally understand you. My hope would have been that tools do what you are trying to do, parse the additional attributes and then take full control over the presentation. However, we got a lot of complaints coming in that error messages in an important secondary tool were missing information. At some point, I decided to extend the messages themselves as that was the only thing I could do from our side.

> Any rollup-specific formatting should be applied once the default log handler is called

Currently, I am not sure if this may be a breaking change for downstream consumers.

> If this isn't possible for some reason, a text property containing the original message should be added to RollupLog.

I would be completely open for this. Maybe call it `originalMessage` to clarify it is about the message attribute.

Not sure I would find time for this soon, a PR would definitely be welcome, though!
